src/js/fluxcontroller.js

```diff
@@ -293,7 +293,9 @@ define(function (require, exports, module) {
             post = action.post,
             parentActionName = actionReceiver.actionName,
             actionName = action.id,
-            actionTitle = "action " + parentActionName;
+            actionTitle = "action " + parentActionName,
+            preferences = this.flux.store("preferences").getState(),
+            checkPost = preferences.get("postConditionsEnabled");
```

Comment:
Is it too much to complain about having to do this lookup every for every action invocation even when not in debug mode?